include/librealsense2/hpp/rs_frame.hpp

```diff
@@ -72,16 +117,37 @@ namespace rs2
             return ss.str();
         }
 
+		/**
+		* Checking if stream profile is set as default
```

Comment:
~~set~~ marked/assigned.
the meaning is that the profile will be selected when the user will request stream configuration using wildcards (RS2_DEPTH, -1,-1,...